srcpkgs/boringtun/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'boringtun'
+pkgname=boringtun
+version=0.2.0
+revision=1
+archs="~*-musl"
```

Comment:
Mark *-musl as broken= instead inside a `case' struct.

archs= is for stuff that cannot under any circumstance work in the arches rejected or can only work in certain arches.

Reply:
Thanks @maxice8